ppr-ui/src/components/unitNotes/UnitNotePanels.vue

```diff
@@ -246,54 +101,68 @@ export default defineComponent({
       default: false
     }
   },
-  setup () {
+  setup (props) {
     const router = useRouter()
 
     const {
       setMhrUnitNoteType
     } = useStore()
 
+    const {
+      isNoticeOfCautionOrRelatedDocType
+    } = useMhrUnitNote()
+
     const localState = reactive({
-      activePanels: []
+      activePanels: [],
+      groupedUnitNotes: computed(() : Array<GroupedNotesIF> => {
```

Comment:
Could this logic or part of it be moved to the unitNotes composable? 
Also, maybe look into `groupBy()` by lodash, I've used it before in the app, and saved me lots of manual work.

Reply:
I will look into moving it into a composable.
I looked into groupBy, I don't think it will work in this case, because the groups are not determined by a common value.